coffee-electron/electron/main.js

```diff
@@ -0,0 +1,39 @@
+const { app, BrowserWindow } = require('electron');
```

Comment:
This whole file ripped straight from this [article](https://blog.logrocket.com/building-a-menu-bar-application-with-electron-and-react/)

Reply:
great find, the article is detailed!